pkgs/development/python-modules/adb-homeassistant/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, libusb1
+, rsa
+, pycryptodome
+, m2crypto
+}:
+buildPythonPackage rec {
+  pname = "adb-homeassistant";
+  version = "1.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "344bcaa0440501c8942fb17f27566439d1f8336c32ef1853750c995d039e58e0";
+  };
+
+  propagatedBuildInputs = [
+    libusb1
+    rsa
+    pycryptodome
+  ];
+
+
+  checkInputs = [ m2crypto ];
+  # disabled as the only test implemented requires m2crypto, resulting in
+  #   undefined symbol: PyFile_Check
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A pure python implementation of the Android ADB and Fastboot protocols";
+    homepage = https://github.com/google/python-adb;
```

Comment:
Do you have an idea where the `adb-homeassistant` repository really is? Is it just a specific commit of `adb`?

Reply:
According to https://github.com/JeffLIrion/python-adb/commit/5949bf432307cbba7128e84d7bc6add7f054a078 the branch of JeffLlrion is the one used for `adb-homeassistant`

Reply:
Then we could set https://github.com/JeffLIrion/python-adb/tree/adb-homeassistant as homepage.